webview/src/experiments/components/table/TableHead.tsx

```diff
@@ -49,7 +49,7 @@ export const TableHead = ({
     if (wrapperHeight) {
       setTableHeadHeight(wrapperHeight)
     }
-  }, [setTableHeadHeight])
+  }, [setTableHeadHeight, headerGroups])
```

Comment:
The alternative is to use `allHeaders` here and wrap its creation in `useMemo` but I think this is fine.